docs/building-images.md

```diff
@@ -212,147 +245,255 @@ kubernetes:
           password: pass
 ```
 
-* `version` - Required; version string of a particular K3s or RKE2 release e.g.`v1.28.0+k3s1` or `v1.28.0+rke2r1`
-* `network` - Required for multi-node clusters, optional for single-node clusters; network configuration for bootstrapping a cluster
-  * `apiVIP` - Required for multi-node clusters, optional for single-node clusters; IP address which will serve as the cluster LoadBalancer (backed by MetalLB)
-  * `apiHost` - Optional; domain address for accessing the cluster
-* `nodes` - Required for multi-node clusters; list of all nodes forming the cluster
-  * `hostname` - Required; a fully qualified domain name (FQDN) which identifies the particular node
-  * `type` - Required; Kubernetes node type - either `server` (for control plane nodes) or `agent` (for worker nodes)
-  * `initializer` - Optional; specifies the cluster initializer. The initializer node is the server node which bootstraps the cluster
-     and allows other nodes to join it. If unset, the first server in the node list will be selected as the initializer.
-* `manifests` - Optional; manifests to apply to the cluster.
-  Can be used separately or in combination with the [configuration directory](#kubernetes-1).
-  * `urls` - Optional; list of HTTP(s) URLs to download the manifests from
-* `helm` - Optional; Helm charts to be deployed to the cluster.
+* `version` - Required; Specifies the version of a particular K3s or RKE2 release (e.g.`v1.28.0+k3s1` or `v1.28.0+rke2r1`)
+* `network` - Required for multi-node clusters, optional for single-node clusters; Defines the network configuration 
+for bootstrapping a cluster.
+  * `apiVIP` - Required for multi-node clusters, optional for single-node clusters; Specifies the IP address which
+  will serve as the cluster LoadBalancer, backed by MetalLB.
+  * `apiHost` - Optional; Specifies the domain address for accessing the cluster.
```

Comment:
I intentionally left the "Optional;" flags here since they are intermixed with required attributes. It felt cleaner to explicitly distinguish them in this case.